test/functional/blockchain.py

```diff
@@ -93,6 +95,22 @@ def _test_gettxoutsetinfo(self):
         assert_equal(res['bestblock'], res3['bestblock'])
         assert_equal(res['hash_serialized_2'], res3['hash_serialized_2'])
 
+    def _test_getblockchaininfo(self):
+        self.log.info("Test getblockchaininfo")
+        node = self.nodes[0]
+        res = node.getblockchaininfo()
+
+        assert_equal(res["chain"], "regtest")
+        assert_equal(res["blocks"], 200)
+        assert_equal(res["headers"], 200)
+        assert_equal(len(res["bestblockhash"]), 64)
+        assert "difficulty" in res
```

Comment:
Instead of multiple `assert str in res`, how about `assert_equal(res.keys(), ...)`? This way if someone adds a new response field or changes the response, it must update this tests too.

So only assert known values, and exact keys.